src/news_app/create_feed.js

```diff
@@ -0,0 +1,48 @@
+export const makeFeed = array => {
+    // CREATE NEWS FEED CONTAINER
+    const newsParent = document.querySelector('.news');
+    const newsContainer = document.createElement('div');
+    newsContainer.classList.add('news-list')
+    newsParent.appendChild(newsContainer)
+    const news = document.querySelector('.news-list');
+    // GENERATE NEWS FEED
+    array.forEach(item => {
+        const {title, author, published, description, source, url, urlToImage} = item;
+        // NEW NEWS DIV
+        const newsDiv = document.createElement('div');
+        newsDiv.classList.add('news-item')
+        // ADD IMAGE
+        const newsPic = document.createElement('img');
```

Comment:
set alt attribute, it is a good practice to think about accessibility and semantic html